python/nano/example/pytorch/inference_pipeline/resnet/README.md

```diff
@@ -56,23 +57,30 @@ python inference_pipeline.py
 ```
 
 ## Results
-
-It will take about 2 minutes to run inference optimization. Then you may find the result for inference as follows:
+It will take about 1 minute to run inference optimization. Then you may find the result for inference as follows:
 ```
 ==========================Optimization Results==========================
-accleration option: original, latency: 54.2669ms, accuracy: 0.9937
-accleration option: fp32_ipex, latency: 40.3075ms, accuracy: 0.9937
-accleration option: bf16_ipex, latency: 115.6182ms, accuracy: 0.9937
-accleration option: int8, latency: 14.4857ms, accuracy: 0.4750
-accleration option: jit_fp32, latency: 39.3361ms, accuracy: 0.9937
-accleration option: jit_fp32_ipex, latency: 39.2949ms, accuracy: 0.9937
-accleration option: jit_fp32_ipex_clast, latency: 24.5715ms, accuracy: 0.9937
-accleration option: openvino_fp32, latency: 14.5771ms, accuracy: 0.9937
-accleration option: openvino_int8, latency: 7.2186ms, accuracy: 0.9937
-accleration option: onnxruntime_fp32, latency: 44.3872ms, accuracy: 0.9937
-accleration option: onnxruntime_int8_qlinear, latency: 10.1866ms, accuracy: 0.9937
-accleration option: onnxruntime_int8_integer, latency: 18.8731ms, accuracy: 0.9875
+ -------------------------------- ---------------------- -------------- ------------
+|             method             |        status        | latency(ms)  |  accuracy  |
+ -------------------------------- ---------------------- -------------- ------------
+|            original            |      successful      |    43.447    |   0.994    |
+|           fp32_ipex            |      successful      |    32.827    |   0.994    |
+|              bf16              |   fail to forward    |     None     |    None    |
+|           bf16_ipex            |        pruned        |   201.702    |    None    |
+|              int8              |      successful      |    10.992    |   0.994    |
+|            jit_fp32            |      successful      |    36.741    |   0.994    |
+|         jit_fp32_ipex          |      successful      |    33.293    |   0.994    |
+|  jit_fp32_ipex_channels_last   |      successful      |    19.523    |   0.994    |
+|         openvino_fp32          |      successful      |    10.51     |   0.994    |
+|         openvino_int8          |      successful      |    6.637     |   0.994    |
+|        onnxruntime_fp32        |      successful      |    20.55     |   0.994    |
+|    onnxruntime_int8_qlinear    |      successful      |     8.15     |   0.994    |
+|    onnxruntime_int8_integer    |   fail to convert    |     None     |    None    |
+ -------------------------------- ---------------------- -------------- ------------
+
+Optimization cost 64.3s at all.
+===========================Stop Optimization===========================
 When accelerator is onnxruntime, the model with minimal latency is:  inc + onnxruntime + qlinear 
```

Comment:
why output this result?

Reply:
This line was supposed to show `get_ best_model` has filter function for accelerator. Or just delete this line and only leave the result by `optimizer.get_best_model(accuracy_criterion=0.05)`